LvivCompany.Bookstore/LvivCompany.Bookstore.Web/Views/BookDetail/Index.cshtml

```diff
@@ -9,12 +10,14 @@
             </div>
             <div class="col-xs-5">
                 <!-- Name -->
-                <h2><b>@Model.Name </b></h2>
+                <h2>@Model.Name</h2>
 
                 <!-- Author -->
-                <label class="title-items"><small>Author   <br></small></label>
-                <span class="inf">@Model.Author<br></span>
-
+                <label class="title-items"><small>Author   </small></label><br>
+                @foreach (var author in @Model.Authors)
+                {
+                <span class="inf">@author.FirstName @author.LastName<br></span>
```

Comment:
Create a property for FullName. Don't use `br` tag unless you want to create a line break between lines in text